### ISSUE TITLE
can not update web papers <app> ( front end or back end) to update code

### ISSUE THREAD

**Issue by chenryadams** [aws-amplify/amplify-hosting#3705]
### Before opening, please confirm:

- [X] I have checked to see if my question is addressed in the [FAQ](https://github.com/aws-amplify/amplify-hosting/blob/master/FAQ.md).
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-hosting/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting questions](https://github.com/aws-amplify/amplify-hosting/blob/master/CONTRIBUTING.md).
- [X] I have removed any sensitive information from my code snippets and submission.

### App Id

you are experiencing issues while creating a new Amplify application

### AWS Region

us-east-1

### Amplify Hosting feature

Backend builds, Build settings, Frontend builds

### Question

can not update web papers <app> ( front end or back end) to update code

**Comment by chenryadams** [comment]
I'm sorry, but I do not speak the front end, back end language that well.  I'm an ole html writer from the web page writing days, whereas, I often used Dream Weaver from Macromedia to get ahead of the game. This full stack stuff confuses the hell out of me.

**Comment by Jay2113** [comment]
Hi @chenryadams 👋 , thanks for raising this!

To better assist you, can you please share your detailed use case and if you are running into any specific issues/errors when hosting your web application using Amplify Hosting?

**Comment by chenryadams** [comment]
Detailed use case?  Well, let's see.   I wrote a html page.  Then , I reviewed the html page looking for errors in my grammar.
Discovering that I wrote a statement incorrectly.  I corrected the statement.  SAVED CORRECTED WEB PAGE & TRIED TO UPDATE IT TO NO AVAIL several times.  Example of line in Web page found incorrect:   <p> C. Adams Henry Web page.  <p>   Example of line within  Web page corrected <p> C. Henry Adams Web page.  <p>

Long story short,  after making correction in a Web pages,  we can update the system with the Web page.
